setup.py

```diff
@@ -5,7 +5,8 @@
     long_description = fh.read()
 
 dask_deps = ["dask>=2.22.0", "distributed>=2.22.0"]
-ray_deps = ["ray[default]>=1.13.0", "pyarrow"]
+# ray==2.5.0 broken: https://github.com/conda-forge/ray-packages-feedstock/issues/100
+ray_deps = ["ray[default]>=1.13.0,!=2.5.0", "pyarrow"]
```

Comment:
We should probably update the min supported version of Ray for the config, as well as exclude the broken release there. Note that MIN_RAY_VERSION is still equal to 1.4.0.

https://github.com/modin-project/modin/blob/13fa0c47178a00f9314a62776396c84db394aae3/modin/config/envvars.py#L113

cc @vnlitvinov, @anmyachev

Reply:
We shouldn't be excluding `2.5.0` in there (Ray itself is working, it's only `ray` command-line tool which is broken on Windows in that release), but should definitely bump the `MIN_RAY_VERSION` indeed.

Reply:
Why should we exclude Ray 2.5.0 here and in other env files?

Reply:
For the consistency of what we test (right now we test 2.5.1 which has released the fix for the issue)

Reply:
I do not understand why we have to stop users from using Ray 2.5.0 with Modin if that Ray version has only ray command-line tool broken?

Reply:
Because we do not actually know if it is working on Windows (because we cannot test this)...

Reply:
Then I do not understand why we do not want to exclude that release from the config.

Reply:
Also, are we excluding the release unconditionally here, but not depending on OSs?

Reply:
I will update `MIN_RAY_VERSION`.